### PR TITLE
Update project files using latest tools

### PR DESCRIPTION
**Pull request by roland-chernov-akvelon**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests if this modifies the UWP implementation?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

This PR updates project files by resaving the solution file using latest Visual Studio for Mac:
- Removes deprecated `AndroidUseLatestPlatformSdk` option.
- Fixes xml files by adding `<U+FEFF>` at the beginning of files.

## Misc

*NOTE*: there is no changes after resaving using Visual Studio for Windows.

### CODE REVIEW

File: Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dhei) [inline Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml]: I am curious what is the change here? Is it a change of UTF-8 to UTF-8 BOM encoding automatically added by Visual Studio? If so, it's best practice not to add them.

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Properties/AndroidManifest.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dhei) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Properties/AndroidManifest.xml]: same question here.

File: Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml (hunk 1 above)
Comment (MatkovIvan) [reply Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml]: @dhei From the desc of the PR:

> Fixes xml files by adding <U+FEFF> at the beginning of files.

All these changes are generated by VS for mac

Comment (dhei) [reply Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml]: Aha, that's okay then. Sorry for failing to read the description.

**Pull request merged**